Signal/src/contact/OWSContactsManager.m

```diff
@@ -571,30 +656,30 @@ - (NSAttributedString *)formattedFullNameForRecipientId:(NSString *)recipientId
             attributes:normalFontAttributes];
 }
 
-- (Contact * _Nullable)contactForPhoneIdentifier:(NSString * _Nullable)identifier {
-    if (!identifier) {
-        return nil;
-    }
-    return self.contactMap[identifier];
+- (nullable SignalAccount *)signalAccountForRecipientId:(nullable NSString *)recipientId
+{
+    OWSAssert(recipientId.length > 0);
```

Comment:
This assert conflicts with the fact that the `recipientId` param  is `nullable`

My hunch is that the param should not be nullable.